### ISSUE TITLE
MailChimp Shutting Down v1 of the API

### ISSUE THREAD

**Issue by gsteve3** [mbergek/autochimp#1]
Hello,

It looks like this is the most recent fork of the autochimp plugin.
We use it across several WordPress sites.
I am trying to find the most maintained fork of it before attempting to update to the latest MailChimp API, which appears to be version 3.0.

At first glance it looks like this fork still uses v1.3 of the MC API.

Is that right?
Do you have plans to update it?

Thanks,

Greg

**Comment by gsteve3** [comment]
Forgot to mention, concern is that MailChimp is deprecating the v1 API next month  - https://mailchi.mp/f2bbb62b3016/were-shutting-down-api-11-12-and-13

**Comment by slateid** [comment]
> Forgot to mention, concern is that MailChimp is deprecating the v1 API next month - https://mailchi.mp/f2bbb62b3016/were-shutting-down-api-11-12-and-13

Any luck in updating this to the version 3 API?

Thanks,
Tom

**Comment by gsteve3** [comment]
@slateid no luck yet - have had other projects come up and had to defer this for a bit but am hoping to work on it over the next week or two. 

Are you using the plugin and just had it stop working?

**Comment by slateid** [comment]
Yep, I had fixed it for the first API deprecation, but now that they are requiring it to be version 3 it has stopped working again.
